Raeng Castagna/week_4/books/main.rb

```diff
@@ -0,0 +1,20 @@
+require 'pry'
+require 'sinatra'
+require 'sinatra/reloader'
+require 'httparty' # add party to require google books api link
+
+book_url = "https://www.googleapis.com/books/v1/volumes?q=title:#{ @book_title }"
```

Comment:
by setting the book_url outside of the method you're searching for the same book every time. hence displaying the same book every time you search.